tests/test_backends.py

```diff
@@ -269,14 +267,13 @@ def test_build_vllm_cmd_with_args_provided(tmp_path: pathlib.Path):
     vllm_args = [
         "--port",
         str(8001),
-        "--model=/path/to/other/model",
         "--distributed-executor-backend",
         "ray",
     ]
     expected_cmd = [
-        sys.executable,
-        "-m",
-        "vllm.entrypoints.openai.api_server",
+        "vllm",
+        "serve",
+        os.fspath(model_path),
```

Comment:
@russellb why was os.fspath(model_path) used here and str(model_path) used for the other tests you changed here?

Reply:
I don't remember. I'm guessing this gets converted to a string automatically so the result is the same. agree that str() would be better for consistency.

Reply:
actually, it was because that's what the code in the vllm backend does

```
    vllm_cmd = [vllm_path if vllm_path else "vllm", "serve", os.fspath(model_path)]
```

which probably come from how the model path was handled in this removed code:

```
    if not contains_argument("--model", vllm_args):
        vllm_cmd.extend(["--model", os.fspath(model_path)])
```

and this is the correct way to convert the type to a string in this case